src/utils/helpers.js

```diff
@@ -63,9 +63,12 @@ export function makeEventFilter(date, { startAccessor, endAccessor }) {
  * Returns a filter function that returns true if an availability should be rendered
  * in the specified date's day column.
  */
-export function makeAvailabilityFilter(date) {
-  return function filterDayAvailabilities(availability) {
-    const AVAILABILITY_WEEK_DAYS = [6, 0, 1, 2, 3, 4, 5];
-    return AVAILABILITY_WEEK_DAYS.indexOf(availability.weekday) === date.getDay();
+export function makeAvailabilityFilter(date, { availabilityStartAccessor, availabilityEndAccessor }) {
```

Comment:
This looks almost exactly the same as `makeEventFilter` now. I think we can consolidate them into a single function? Along the lines of `makeEventOrAvailabilityFilter`.

Reply:
I tried consolidating, but could only call the consolidated version one time. The `startAccessor` and `endAccessor` are undefined if the `makeEventOrAvailabilityFilter` is called a second time. Might need a pointer here.

Reply:
Actually, I think I see what's going on and can fix it.